src/python/arcor2_arserver/rpc/project.py

```diff
@@ -1183,6 +1283,9 @@ async def remove_logic_item_cb(req: srpc.p.RemoveLogicItem.Request, ui: WsClient
     assert glob.PROJECT
     assert glob.SCENE
 
+    if not await glob.LOCK.is_write_locked(req.args.logic_item_id, "ui.owner"):
```

Comment:
S tím si ale aktuálně `get_root_id` neporadí, ne?

Reply:
Pokud si dobře vzpomínám, tak logic item je vlastně akce (id=act_...) ještě to otestuju, až bude podpořené zamykání akcí z editoru

Reply:
Tak máte pravdu, opravdu se jedná o jiný objekt... Zatím zámky zakomentuji a ještě proberu, jak to zvládne zamykat editor

Reply:
`LogitItem` je ten propoj mezi akcemi, má property `start` (id akce kde začíná) a `end` (id akce kde končí).

Reply:
Přidáno zamykání odemčených objektů. Toto bude fungovat i v případě, že si editor zamkne jednu akci, rozpojí logiku a pak jí zapojí jinak.